astroquery/alma/tests/test_alma_remote.py

```diff
@@ -14,6 +14,7 @@
 from pyvo.dal.exceptions import DALOverflowWarning
 
 from astroquery.exceptions import CorruptDataWarning
+import astroquery
 from .. import Alma
```

Comment:
hmm, could we instead directly import the function? (though I kind of agree that in some cases it's good not doing that an instead be super explicit at use time (but even then, usage should be from the API namespace, not at the code's location (this could be a generic TODO to clean up all the "core"s from the astroquery tests).

```suggestion
from astroquery.alma import Alma, get_enhanced_table
```